Editor/Window/ManagedEC2Page.cs

```diff
@@ -93,6 +95,30 @@ public ManagedEC2Page(VisualElement container, StateManager stateManager)
             UpdateGUI();
         }
 
+        private void UpdateDeploymentSettings()
+        {
+            ManagedEC2Deployment ec2Deployment;
+            if (_stateManager.IsBootstrapped)
+            {
+                _deploymentSettings.Refresh();
+                _deploymentSettings.Restore();
+                _ec2Deployment.UpdateModelFromParameters(new ManagedEC2FleetParameters
+                {
+                    GameName = _deploymentSettings.GameName ?? Application.productName,
+                    FleetName = _deploymentSettings.FleetName ?? $"{Application.productName}-ManagedFleet",
+                    LaunchParameters = _deploymentSettings.LaunchParameters ?? $"",
+                    BuildName = _deploymentSettings.BuildName ??
+                                $"{Application.productName}-{_deploymentSettings.ScenarioName.Replace(" ", "_")}-Build",
+                    GameServerFile = _deploymentSettings.BuildFilePath,
+                    GameServerFolder = _deploymentSettings.BuildFolderPath,
+                    OperatingSystem = OperatingSystem.FindValue(_deploymentSettings.BuildOperatingSystem) ??
+                                      OperatingSystem.AMAZON_LINUX_2
+                });
+            }
```

Comment:
This code duplicates lines 33-44, can we create a utility `GetManagedEc2FleetParameters(DeploymentSettings deploymentSettings)` that returns a `ManagedEC2FleetParameters` instead